pkg/server/handler.go

```diff
@@ -43,18 +43,40 @@ func NewMsgHandler(logger logging.Logger, amqp network.IAmqpReceiver, thingContr
 
 // Start starts to listen messages
 func (mc *MsgHandler) Start(started chan bool) {
```

Comment:
Since we're returning errors now, shouldn't we add this? :point_down: 

```suggestion
func (mc *MsgHandler) Start(started chan bool) err {
```

Reply:
Hmmmm, I'd like to see what @netoax and @esdrasjnr think abou this. I don't know a lot about error handling in `go routines`. I'll dig a little deeper, but I do understand your POV and (from what I know, now) it should be fine.

Reply:
I think in this case, as the control is redirected to [here](https://github.com/CESARBR/knot-babeltower/blob/master/cmd/main.go#L87), we don't need to worry about this error too much. At least for me, it's ok just logging it.